drive-stack/conUDS/src/arguments.rs

```diff
@@ -35,17 +39,31 @@ pub enum ArgSubCommands {
     BootloaderDownload(SubArgBootloaderDownload),
     ReadDID(SubArgReadDID),
     NVMHardReset(SubArgNVMHardReset),
+    Batch(SubArgBatch),
 }
 
 /// Download an application to an ECU
 #[derive(Debug, FromArgs)]
 #[argh(subcommand, name = "download")]
 pub struct SubArgDownload {
+    /// only perform download if there is a CRC mismatch
+    #[argh(switch, short = 'c')]
+    pub verify_crc: bool,
```

Comment:
call this `force` and default to true

Reply:
cant default to true, so changed to no skip